include/config.h

```diff
@@ -81,4 +71,11 @@
 #define SUMMARY_SCREEN_NATURE_COLORS TRUE    // If TRUE, nature-based stat boosts and reductions will be red and blue in the summary screen.
 #define HQ_RANDOM                    TRUE    // If TRUE, replaces the default RNG with an implementation of SFC32 RNG. May break code that relies on RNG.
 
+// Measurement system constants to be used for UNITS
+#define UNITS_IMPERIAL 0 // Inches, feet, pounds
+#define UNITS_METRIC 1 // meters, kilograms
```

Comment:
I'd expect alignment here to apply to both the values of the defines and the comments

Reply:
solved with https://github.com/rh-hideout/pokeemerald-expansion/pull/4183/commits/fd0ab7664345612537c62e1e878263f02568fbfa